.github/CODEOWNERS

```diff
@@ -43,10 +43,14 @@
 
 # System semantic conventions
 /docs/process/                    @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers @open-telemetry/semconv-security-approvers
-/docs/system/                     @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers
 /docs/resource/host.md            @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers
+/docs/resource/process.md         @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers
+/docs/system/                     @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers
+/model/disk/                      @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers
 /model/host/                      @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers
-/model/process/                    @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers @open-telemetry/semconv-security-approvers
+/model/network/                   @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers
```

Comment:
I would expect this might need other area experts as well. Not sure which other areas this covers though.

Reply:
Network is already defined in the HTTP section and I didn't notice. I'll fix that up.

Reply:
Fixed in [8a5524b](https://github.com/open-telemetry/semantic-conventions/pull/1623/commits/8a5524b663878271c14d5a441efae4415fe2295e)